src/test/javascript/spec/component/team/teams-export-button.component.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { TeamService } from 'app/exercises/shared/team/team.service';
+import { TeamsExportButtonComponent } from 'app/exercises/shared/team/teams-import-dialog/teams-export-button.component';
+import * as chai from 'chai';
+import { JhiAlertService } from 'ng-jhipster';
+import { SinonSpy, SinonStub, spy, stub } from 'sinon';
+import * as sinonChai from 'sinon-chai';
+import { mockTeams } from '../../helpers/mocks/service/mock-team.service';
+import { config } from './teams-import-dialog.component.spec';
```

Comment:
I would suggest not doing this. Tests should not depend on the config of other tests. Do you really need all the dependencies here too?